hooks/useFeedScrollPosition.ts

```diff
@@ -1,14 +1,14 @@
 import { useState, useEffect } from 'react';
 
-function useFeedScrollPosition() {
+function useFeedScrollPosition(feedContainerName: string = 'feed-container') {
```

Comment:
same question here, why do we need a `feedContainerName` prop?
- when is `feedContainerName` not `feed-container`???